src/shared/components/Button.jsx

```diff
@@ -71,6 +71,8 @@ export default function Button({
       onClick={onClick}
       onMouseDown={onMouseDown}
       type={type}
+      tabIndex={0}
```

Comment:
again, I don't think it's necessary to force a "translation" of {0} to ="0" when you already know it will be 0. tabIndex="0" should be alright